src/briefcase/commands/config.py

```diff
@@ -0,0 +1,136 @@
+from pathlib import Path
+
+import tomli
```

Comment:
If you're on Python 3.11 or greater, `tomli` isn't installed; you can use `tomllib` from the standard library. See `base.py` for the import pattern that is used.